chain/store/store.go

```diff
@@ -579,3 +584,130 @@ func (cs *ChainStore) CalcNextDonation(height uint32) (*Donation, error) {
 
 	return d, nil
 }
+
+func (cs *ChainStore) GetStateRoots(fromHeight, toHeight uint32) ([]Uint256, error) {
```

Comment:
We probably need to check whether `fromHeight <= toHeight` is satisfied

Reply:
"for i := fromHeight; i <= toHeight; i++ " do it.